src/directives/unsafe-html.ts

```diff
@@ -47,7 +47,7 @@ export const unsafeHTML = directive((value: any) => (part: Part): void => {
   }
 
   const template = document.createElement('template');
-  template.innerHTML = value;
+  template.innerHTML = String(value);
```

Comment:
I'd cast instead. The `innerHTML` setter does toString if necessary.

Reply:
Done